modules/navier_stokes/src/fvbcs/INSFVSwitchableOutletPressureBC.C

```diff
@@ -0,0 +1,57 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "INSFVSwitchableOutletPressureBC.h"
+#include "INSFVPressureVariable.h"
+#include "Function.h"
+
+registerMooseObject("NavierStokesApp", INSFVSwitchableOutletPressureBC);
+
+InputParameters
+INSFVSwitchableOutletPressureBC::validParams()
+{
+  InputParameters params = INSFVOutletPressureBC::validParams();
+
+  params.addClassDescription("Adds switchable pressure-outlet boundary condition");
+
+  params.addParam<bool>("switch", true, "Switch on (true) / off (false) for boundary condition.");
+  params.declareControllable("switch");
+
+  params.addParam<Real>("face_limiter", 1.0, "Interpolation face flux limiter.");
```

Comment:
Is "interpolation" an accurate descriptor given the boundary nature of this?

Reply:
Thanks. Removed 'Interpolation'